target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -183,6 +183,9 @@ fonera20n)
 	set_usb_led "$board:orange:usb"
 	set_wifi_led "$board:orange:wifi"
 	;;
+gb-pc1)
```

Comment:
and I pushed this too ... still not sure what happened.
gb-pc1)
	ucidef_set_led_netdev "lan" "lan" "$board:green:lan" "eth0.1"
	ucidef_set_led_netdev "wan" "wan" "$board:green:wan" "eth0.2"
	;;